docs/user-guide/authenticating-with-client-certificates.md

```diff
@@ -84,3 +84,16 @@ curl --cert /path/to/cert.pem --key /path/to/key.pem https://api-mediation-layer
 ```
 Your Zowe instance is configured to accept x.509 client certificates authentication.
 
+**Java sample application**
```

Comment:
should it mention that it requires java 17?

Reply:
Is there any reason why it would not run with 8?

Reply:
I see, there are 'var's in the code. I will add the prerequisite.